kuma/static/styles/components/home/column-callout.scss

```diff
@@ -239,36 +238,51 @@ $saucelabs-callout-image-spacing: 129px + 20px;
 }
 
 /*
-larger teaser image gets hidden and shown at a different rate than the usual callouts
-- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - */
+larger teaser image gets hidden
+- - - - - - - - - - - - - - - - */
 
 @media #{$mq-mobile-only} {
-    .callout-saucelabs {
+    .callout-standard {
         a:after {
             display: block;
         }
 
         span {
-            @include bidi(((padding, $grid-spacing ($saucelabs-callout-image-spacing + $grid-spacing) $grid-spacing $grid-spacing, $grid-spacing $grid-spacing $grid-spacing ($saucelabs-callout-image-spacing + $grid-spacing)),));
+            @include bidi(((padding, $grid-spacing ($standard-callout-image-spacing + $grid-spacing) $grid-spacing $grid-spacing, $grid-spacing $grid-spacing $grid-spacing ($standard-callout-image-spacing + $grid-spacing)),));
         }
     }
 }
 
 @media #{$mq-large-desktop-and-up} {
-    .callout-saucelabs {
+    .callout-standard {
         a:after {
             display: block;
         }
 
         span {
-            @include bidi(((padding, $grid-spacing ($saucelabs-callout-image-spacing + $grid-spacing) $grid-spacing $grid-spacing, $grid-spacing $grid-spacing $grid-spacing ($saucelabs-callout-image-spacing + $grid-spacing)),));
+            @include bidi(((padding, $grid-spacing ($standard-callout-image-spacing + $grid-spacing) $grid-spacing $grid-spacing, $grid-spacing $grid-spacing $grid-spacing ($standard-callout-image-spacing + $grid-spacing)),));
         }
     }
 }
 
 @media (-webkit-min-device-pixel-ratio: 1.5), (min--moz-device-pixel-ratio: 1.5), (-o-min-device-pixel-ratio: 3/2), (min-resolution: 1.5dppx) {
-    .callout-saucelabs a:after {
-        background-image: url($path-to-embedded-images + 'promos/sauce-labs-home-high-res.png');
+    .callout-standard a:after {
         background-size: 129px 140px;
     }
 }
+
+/*
+Survey callout background image
+-------------------------------------------------------------- */
+
+.callout-survey {
+    &:after {
+        background-image: url($path-to-embedded-images + 'promos/survey.svg');
+    }
+}
+
+@media (-webkit-min-device-pixel-ratio: 1.5), (min--moz-device-pixel-ratio: 1.5), (-o-min-device-pixel-ratio: 3/2), (min-resolution: 1.5dppx) {
+    .callout-survey a:after {
+        background-image: url($path-to-embedded-images + 'promos/survey.svg');
```

Comment:
I think I led you down the wrong path. Here's what the banner looks like now:

![screenshot 2018-05-29 07 55 45](https://user-images.githubusercontent.com/286017/40660531-e2a99f20-6316-11e8-8068-afd5a2ce9805.png)

Maybe we should return to ``callout-standard`` for this rule.

Reply:
Interesting that you see that, I guess I did not tests again after rebasing. I had a hell of a time rebasing and some thing clearly got lost in translation. I will resolve this issue. Having been without power for the entire day :-/ I will take care of this first thing tomorrow. Thanks for highlighting this.

Reply:
Sorry about your rough day! If it helps, here's the branch before the latest rebase:

https://github.com/jwhitlock/kuma/tree/survey-banner-1462835

Reply:
Much better!

<img width="450" alt="screenshot 2018-05-30 08 51 57" src="https://user-images.githubusercontent.com/286017/40724472-bf6f08d0-63e6-11e8-8778-ff53652b3554.png">